todoman/model.py

```diff
@@ -474,7 +474,9 @@ def create_tables(self):
                 "uid" TEXT,
                 "summary" TEXT,
                 "due" INTEGER,
+                "due_dt" INTEGER,
```

Comment:
You'll need to push `SCHEMA_VERSION` so that existing installations re-run cache table creation.